Demo/Demo/SwiftUIComponents/SwiftUINativeCheckoutDemo.swift

```diff
@@ -24,12 +74,40 @@ struct SwiftUINativeCheckoutDemo: View {
             }
         }
     }
+
+    func loadingView(_ content: String) -> some View {
+        VStack(spacing: 16) {
+            ProgressView(content)
+                .progressViewStyle(CircularProgressViewStyle())
+        }
+    }
+
+    func errorView(_ message: String) -> some View {
+        VStack(spacing: 16) {
+            Text(message)
+        }
+    }
+
+    func startNativeCheckout() {
+        switch checkoutTypeSelection {
+        case .order:
+            viewModel.checkoutWithOrder()
+        case .orderID:
+            viewModel.checkoutWithOrderID()
+        case .billingAgreement:
+            viewModel.checkoutWithBillingAgreement()
+        case .baWithoutPurchase:
+            viewModel.checkoutBAWithoutPurchase()
+        case .vault:
+            viewModel.checkoutWithVault()
```

Comment:
Same comment as below, re: removing unneeded Demo types as discussed in standup.